tests/functional/tests/configure/ceph.py

```diff
@@ -53,16 +53,17 @@ def _get_osd_map() -> Dict[str, List[str]]:
 
 def _set_osd_device_class(osds: List[str], device_class: str):
     """Set device class for osds."""
+    unit = "ceph-mon/0"
```

Comment:
this is a nit pick because this likely does not affect CI, but rather than hard-coding unit name couldn't we just pick random unit from ceph-mon application? Something like:
```python
unit = zaza.model.get_application("ovn-central").units[0].entity_id
```
(same for other parts that use hardcoded unit name)

Reply:
I was going to push this, when I realized that we are using `self.model.units["<app-name>/0"]` everywhere. I know that this is configuration, but I doubt that it will be reused in other places. At the same time the approach with constant is faster.

It might be a good idea to use it in all configurations (maybe also in tests), but don't think it's worth running the functional test again right now.